docs/getting-started/cases-req.asciidoc

```diff
@@ -8,4 +8,36 @@ For more information, see
 {kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges].
 
 NOTE: To send cases to external systems, you need the
-https://www.elastic.co/subscriptions[appropriate license].
\ No newline at end of file
+https://www.elastic.co/subscriptions[appropriate license].
+
+You can create roles and define feature privileges at different levels to manage feature access in {kib}. {kib} privileges grant access to features within a specified {kib} space and full or partial access can be given.
+
+When setting up feature access to cases, make sure to configure the following:
+
+- Base privileges for the {es-sec} solution
+- Sub-feature privileges for the Cases feature
+- Privileges for actions and connectors
+
+NOTE: Privileges set at the sub-feature level will override privileges set at the base level for the {es-sec} solution. For example, if a user's role has `All` privileges for the {es-sec} solution and `None` selected for the Cases feature, they'll have access to all features within the {es-sec} app except for cases.
```

Comment:
This is very helpful. Should we also note that if the user does not customize the sub-feature privilege the user will be granted the same access to cases? For example if they have All for Security Solution and have not customized the cases sub-feature (the toggle is disabled), the user will have All for cases.

Reply:
Agree that this is a good example to include. I'll likely insert it right after the line I add about the default state of case privs.